src/App.css

```diff
@@ -1,38 +1,347 @@
+/* imports */
+
+@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@400;500;600;700&family=Source+Sans+Pro:wght@400;600;700&display=swap');
```

Comment:
Interesting, I always import google fonts in the html head but I'm not sure what the difference is (if any)